server.rb

```diff
@@ -17,16 +17,22 @@
 EasyTranslate.api_key= ENV['TRANSLATE_API_KEY']
 @@users = {}
 
-def language(id)
-  languages = EasyTranslate::LANGUAGES.invert
-  choice = @@users[id].downcase
-  languages.each do |key, value|
-    if key == choice
-      return {to: value.to_sym}
-    end
+# Original Way I wrote the method
+  # def language(id)
+  #   languages = EasyTranslate::LANGUAGES.invert
+  #   choice = @@users[id][:language].downcase
+  #   languages.each do |key, value|
+  #     if key == choice
+  #       return {to: value.to_sym}
+  #     end
+  #   end
+  #   return nil
+  # end
+
+# Cleaner way to write the language method
+  def language(id)
+    EasyTranslate::LANGUAGES.invert[@@users[id][:language].downcase]
```

Comment:
this can be clean if the purpose is well understood. it's a function that takes something and returns something. it becomes a blackbox.

@PrimeTimeTran you can make it clear by stating assumptions of what `@@users` is like and what EasyTranslate::Languages.invert is like. the old way of returning a hash with `{to: }` key was confusing and it took 10 lines of code.